code/nodes/saveReport/reportlab/utils.q

```diff
@@ -23,27 +26,28 @@ saveReport.i.printDescripTab:{[tab]
 
 // @kind function
 // @category saveReportUtility
-// @fileoverview Convert kdb table to printable format
+// @desc Convert kdb table to printable format
 // @param pdf {<} PDF gen module used
-// @param f   {int} The placement height from the bottom of the page
-// @param tab {tab} kdb table to be converted
+// @param f {int} The placement height from the bottom of the page
+// @param tab {table} kdb table to be converted
 // @return {int} The placement height from the bottom of the page
 saveReport.i.printKDBTable:{[pdf;f;tab]
   dd:{(,'/)string(key x;count[x]#" ";count[x]#"=";count[x]#" ";value x)}tab;
-  cntf:first[count dd]{[m;h;s]ff:saveReport.i.text[m;h 0;15;s h 1;"Helvetica";11];
+  cntf:first[count dd]{[m;h;s]ff:saveReport.i.text[m;h 0;15;s h 1;
+    "Helvetica";11];
     (ff;1+h 1)}[pdf;;dd]/(f-5;0);
   first cntf
   }
 
 // @kind function
 // @category saveReportUtility
-// @fileoverview
-// @param m   {<} pdf gen module used
-// @param h   {int} the placement height from the bottom of the page
-// @param i   {int} how far below is the text
-// @param txt {str} text to include
-// @param f   {int} font size
-// @param s   {int} font size
+// @desc Add text to report
+// @param m {<} pdf gen module used
```

Comment:
All the below descriptions can be capitalized

Reply:
Descriptions have been capitalized